doc/neps/nep-0018-array-function-protocol.rst

```diff
@@ -669,63 +777,6 @@ However, to work well this would require the possibility of implementing
 *some* but not all functions with ``__array_function__``, e.g., as described
 in the next section.
 
-Coercion to a NumPy array as a catch-all fallback
```

Comment:
I'm not sure we should delete this whole section; I'd rather add that with `__numpy_implementation__` there seems to be no need for it any more.

Reply:
+1 for not changing history -- I'd prefer to keep this section with a disclaimer that it's been replaced by section X.

Reply:
I'll leave around some of this section in abbreviated form, but I've been trying to write this document first and foremost for future readers. Preserving history is a secondary concern at best -- that's what git is for.

Ultimately, I think readers will be interested in understanding the system we wrote, not the system we originally planned to write.

Reply:
I think this should maybe be put in a `Rejected alternatives` section like upstream PEPs have

Reply:
I added back in most of this section, but rewritten in the context of the current proposal.